src/Upgrade/Application/Dto/StepsResponseDto.php

```diff
@@ -114,6 +119,24 @@ public function getComposerLockDiff(): ?ComposerLockDiffDto
         return $this->composerLockDiffDto;
     }
 
+    /**
+     * @return \Upgrade\Application\Dto\IntegratorResponseDto|null
+     */
+    public function getIntegratorResponseDto(): ?IntegratorResponseDto
+    {
+        return $this->integratorResponseDto;
+    }
+
+    /**
+     * @param \Upgrade\Application\Dto\IntegratorResponseDto|null $integratorResponseDto
+     *
+     * @return void
+     */
+    public function setIntegratorResponseDto(?IntegratorResponseDto $integratorResponseDto): void
+    {
+        $this->integratorResponseDto = $integratorResponseDto;
```

Comment:
add `return $this` to have consistence setters in this class

Reply:
Thanks ++